### PR TITLE
Update appdev-deploy.md

### PR DESCRIPTION
**Pull request by solipet**
Moved the direction to recheck NR in a hour after the scaling in of old servers.
Copied to note to do a regular merge to the production merge.

### CODE REVIEW

File: _articles/appdev-deploy.md [hunk 1]
```diff
@@ -142,7 +144,7 @@ Scheduled for every other Wednesday
 ## Production
 Scheduled for every other Thursday
 
-1. Merge the production promotion pull request
+1. Merge the production promotion pull request (**NOT** a squashed merge, just a normal merge)
```
Comment (zachmargolis) [inline _articles/appdev-deploy.md:147]: 💯

**Review by zachmargolis — APPROVED** [review]
LGTM! thanks for these!

File: _articles/appdev-deploy.md [hunk 2]
```diff
@@ -11,8 +11,10 @@ This guide assumes that:
 - You have a [GPG key set up with GitHub](https://help.github.com/en/github/authenticating-to-github/adding-a-new-gpg-key-to-your-github-account) (for signing commits)
 - You have [set up `aws-vault`]({{site.baseurl}}/articles/infrastructure-setting-up-aws-vault.html), and have can SSH (via `ssm-instance`) in to our production environment
 
+Note: it is a good idea to make sure you have the latest pulled down from identity-devops - lots of goood improvements all the time!
```
Comment (zachmargolis) [inline _articles/appdev-deploy.md:14]: Maybe we make this more automatic? `cd identity-devops; git pull` in the step-by-steps?

File: _articles/appdev-deploy.md (hunk 1 above)
Comment (mitchellhenke) [reply _articles/appdev-deploy.md:147]: thank you 😓

**Review by mitchellhenke — APPROVED** [review]

File: _articles/appdev-deploy.md (hunk 2 above)
Comment (solipet) [reply _articles/appdev-deploy.md:14]: added a `git pull` to the first time we cd into identity-devops

**Pull request merged**